.github/workflows/publish-docs.yml

```diff
@@ -34,6 +34,11 @@ jobs:
         with:
           jvm: temurin:1.11
 
+      - name: Set  up Akka resolver
```

Comment:
Is this one missing the actual env var setup, or is it somehow inherited?

Reply:
It is not, good catch. Adding now.